csvcubed/csvcubed/models/cube/qb/catalog.py

```diff
@@ -29,11 +29,17 @@ class CatalogMetadata(CatalogMetadataBase, UriIdentifiable):
     license_uri: Optional[str] = field(default=None, repr=False)
     public_contact_point_uri: Optional[str] = field(default=None, repr=False)
     uri_safe_identifier_override: Optional[str] = field(default=None, repr=False)
+    spatial_bound_uri: Optional[str] = field(default=None, repr=False)
+    temporal_bound_uri: Optional[str] = field(default=None, repr=False)
 
     _creator_uri_validator = validate_uri("creator_uri", is_optional=True)
     _publisher_uri_validator = validate_uri("publisher_uri", is_optional=True)
     _landing_page_uris_validator = validate_uris_in_list("landing_page_uris", is_optional=True)
     _license_uri_validator = validate_uri("license_uri", is_optional=True)
+    _spatial_bound_uri_validator = validate_uri("spatial_bound_uri", is_optional=True)
```

Comment:
These spatial and temporal bounds are technically a new feature which will require other changes in the underlying csvcubed models. @canwaf do we want to drop these for now and pick them up in a separate issue?

Reply:
Confirmed with canwaf - commented out.